legend-depot-pure-model-context/src/main/java/org/finos/legend/depot/services/pure/model/context/PureModelContextServiceImpl.java

```diff
@@ -71,6 +72,14 @@ public PureModelContextData getPureModelContextData(String groupId, String artif
         });
     }
 
+    @Override
+    public PureModelContextData getDependenciesPureModelContextData(List<ProjectVersion> projectDependencies, String clientVersion, boolean transitive, boolean includeOrigin)
```

Comment:
can we rename this to getPureModelContextData(List<ProjectVersion>..) it becomes self explanatory

Reply:
done